search-api/src/search_api/models/document.py

```diff
@@ -0,0 +1,66 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Table for storing document details."""
+from enum import Enum
+
+from .db import db
+
+
+class Document(db.Model):
+    """Used to hold the documents requested in an access request."""
+
+    class DocumentTypes(str, Enum):
```

Comment:
I think this should use our new BaseEnum

Reply:
Done